packages/smooth_app/lib/cards/product_cards/knowledge_panels/knowledge_panel_title_card.dart

```diff
@@ -83,6 +88,21 @@ class KnowledgePanelTitleCard extends StatelessWidget {
   }
 
   Color? _getColorFromEvaluation(Evaluation evaluation) {
+    switch (evaluation) {
+      case Evaluation.BAD:
+        return RED_COLOR;
+      case Evaluation.NEUTRAL:
+        return PRIMARY_GREY_COLOR;
+      case Evaluation.AVERAGE:
+        return LIGHT_ORANGE_COLOR;
+      case Evaluation.GOOD:
+        return LIGHT_GREEN_COLOR;
+      case Evaluation.UNKNOWN:
+        return null;
+    }
+  }
+
+  Color? _getColorFromEvaluationDarkMode(Evaluation evaluation) {
```

Comment:
Never `null`: `Color` instead of `Color?`.

Reply:
Replaced `Color?` with `Color`.